### ISSUE TITLE
The button beside session button "join and start collaboration" have no tool tip in display when keyboard focus is received to each.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#111017]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#Win32;#DesktopApp;#A11y_VSCode;#WCAG1.3.3;#A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;#Keyboard;

### **Environment Details:**
Application Name:  Visual Studio Code
Application version: 1.49.0
Windows Version: Win10 

### **Repro Steps:**

1. Launch VS Code
2. Navigate to the “sign-in” button using tab .
3. Tab to share button and use enter key.
4. Tab to Session Details dropdown 
5. Use the tab key to navigate to the button "join and start collaboration"

### **Actual:**
On navigating to the buttons using tab key "join and start collaboration" there is no tooltip displayed for them, which is available on mouse hover.

### **Same issue is observed in the following page:**

1. no collaboration Planned Sessions
2. no collaboration Contacts
3. with collaboration Session Details
4. with collaboration  Contacts
5. Audio Call-End Audio Call
6. Contacts-Change Your Own availability Status
7. Contacts-Collaborate with  a recent contact
8. Contacts-Collaborate with suggested contact

### **Expected:**
On the tab navigation to each of the button, the tooltip should be displayed.
On escape the tooltip should be dismissed.

### **User Impact:**
Keyboard users will not know what is the use of the button through the icon for the button.

### Attachment for Reference:
[1214890_BM_VisualStudioCodeLiveShare-Sep2020_Nocollaboration_SessionDetail_Keyboard .zip](https://github.com/microsoft/vscode/files/5572763/1214890_BM_VisualStudioCodeLiveShare-Sep2020_Nocollaboration_SessionDetail_Keyboard.zip)

**Comment by RMacfarlane** [comment]
Please open this issue on the LiveShare extension
